src/br/bsb/liberdade/baas/db.clj

```diff
@@ -22,12 +20,19 @@
 ; # MIGRATE UP #
 ; ##############
 
+(defn spy [x]
+  (println x)
+  x)
+
+(defn get-opp [k m]
```

Comment:
Is there a Clojure core function that does this already?

Reply:
The function that uses this could actually be rewritten as

```
(-> (strint/strint "..." {"migration" migration})
    execute-query
    first
    (get (keyword "count (*)"))
    pos?)
```

This way, there is no need to reinvent the `get` function